drake/doc/ubuntu.rst

```diff
@@ -49,3 +54,9 @@ Update the symbolic link in MATLAB to point to the version that was installed ea
     sudo ln -s /usr/lib/gcc/x86_64-linux-gnu/4.9/libstdc++.so libstdc++.so.6
 
 When you are done with these platform-specific steps, return to :doc:`from_source` to complete and test your installation.
+
+Note that when you run drake commands from now on (including the
+ones in the linked instructions, such as `make` or `make test`),
+you must always precede them with
+`env CXX=g++-4.9 CC=gcc-4.9 PATH=/usr/lib/ccache:$PATH`,
```

Comment:
Huh.  I tested leaving it out, and saw 'gcc 4.8' during the superbuild from one of the dependencies.  Would it make more sense to leave this paragraph in place, while we debug that defect?